app/script/view_model/content/TitleBarViewModel.js

```diff
@@ -114,16 +116,24 @@ z.viewModel.content.TitleBarViewModel = class TitleBarViewModel {
     this.showDetails();
   }
 
-  onMouseDown() {
-    this.isDragged = this.isMacDesktop;
+  onMouseDown(_, event) {
+    if (this.isMacDesktop) {
+      this.isDragged = true;
+      this.startX = event.screenX;
+      this.startY = event.screenY;
+    }
   }
 
-  onMouseMove() {
-    this.isMoved = this.isDragged;
+  onMouseMove(_, event) {
+    if (this.isDragged && !this.isMoved) {
+      const diffX = Math.abs(event.screenX - this.startX);
+      const diffY = Math.abs(event.screenY - this.startY);
+      this.isMoved = diffX > 2 || diffY > 2;
```

Comment:
move the threshold to a CONST:

`````
static get CONFIG() {
  return {
    DRAG_THRESHOLD: 2,
  };
}

this.isMoved = differenceX > TitleBarViewModel.CONFIG.DRAG_THRESHOLD;
````

Also characters are free diff-> difference